build/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM registry.access.redhat.com/rhel7-atomic
+
+LABEL name="OpenStorage Operator" \
+      vendor="" \
+      version="v0.1.0" \
+      release="1" \
+      summary="OpenStorage Operator" \
+      description="This operator will deploy 3 instances of OpenStorage operator in the cluster."
```

Comment:
I think the `3 instances` should be removed from the description here. This will be the docker image description which doesn't need that detail. It can be in the operator metadata.